applications/teletype/src/teletype_renderer.erl

```diff
@@ -218,7 +218,7 @@ log_warnings(Ws, Template) ->
 -spec log_infos(string(), string(), [info()], binary()) -> ['ok'].
```

Comment:
Return type is now a lie. Also to help the VM discard unused memory early, make this function return `ok`?